### DISCUSSION TITLE
Field condition with collection value

### DISCUSSION THREAD

**Discussion by crackAT** [statamic/cms#11011] [Q&A]
Hi all, 

i don't know what i have to use to show fields in the asset blueprint for a specific collection. 

So what i want to do: I have different collections and all collections are using the same asset container but some collection need different fields in the asset blueprint and i want to hide or show fields they are not needed. 

<img width="918" alt="image" src="https://github.com/user-attachments/assets/c9a66556-da04-4dd0-a900-95293dfa77bb">

any idea. 

Thanks

**Comment by jasonvarga** [comment]
At the moment you can't do that, however since https://github.com/statamic/cms/pull/10588 it should be very simple to add.
